static/css/accounts/edit_tip.css

```diff
@@ -0,0 +1,5 @@
+#edit_tip {
+    align-content: center;
+    margin-top: 5%;
+    margin-bottom: auto;
+}
```

Comment:
missing line

Reply:
In css files the formating is without a new line I think

Reply:
I see that all the files in our project have a new line at the end of the file

> In css files the formating is without a new line I think